packages/eslint-config-recommended/docs/base_eslintrc.cjs

```diff
@@ -0,0 +1,101 @@
+import {
+  TSLINT_ORIGINATED_GENERALLY_APPLICABLE_RULES,
+  TSLINT_ORIGINATED_TS_SPECIFIC_RULES,
+} from './tslint-to-eslint-mapping'
+
+module.exports = {
+  /*
+   * note: "root" is explicitly not set: this has to be defined by the consuming eslintrc declaration
+   */
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint',
+    'eslint-plugin-import',
+    'eslint-plugin-prefer-arrow',
+    'jsdoc',
+  ],
+  overrides: [
+    /*
+     * Generally applicable JS + TS rules (based on former TSLint config)
+     */
+    {
+      files: ['*.js', '*.cjs', '*.mjs', '*.jsx', '*.ts', '*.cts', '*.mts', '*.tsx'],
+      extends: [
+        'eslint:recommended', // catching common JS problems
+      ],
+      rules: {
+        ...TSLINT_ORIGINATED_GENERALLY_APPLICABLE_RULES,
+
+        /*
+         * Rule overrides (or additional rules)
+         */
+        'no-empty': 'off', // TSLint: "no-empty" (override "eslint/recommended")
+        'no-fallthrough': 'off', // override "eslint:recommended" rule
+
+        /*
+         * rules that are covered by ESLint plugins
+         */
+        'import/no-deprecated': 'warn', // TSLint: "deprecation"
+        'import/no-internal-modules': [ // TSLint: "no-submodule-imports"
+          'error',
+          {
+            allow: [
+              '@angular/*',
```

Comment:
this is very project specific and should be added in the project configuration

Reply:
```json
 'import/no-internal-modules': [ // TSLint: "no-submodule-imports"
          'error',
          {
            allow: [
              '@angular/*',
              '@apollo/*',
              '@shiftcode/iac-helper/*',
              'app/',
              'aws-sdk/*',
              'aws-cdk-lib/*',
              'aws-serverless-express/middleware/*',
              'core-js/*',
              'date-fns/esm/locale/*',
              'fs/*',
              'zone.js/*',
            ],
          },
        ],
```